src/components/nav-drawer/themes/shared/header-item/header-item.material.scss

```diff
@@ -0,0 +1,10 @@
+@use 'styles/utilities' as *;
+@use '../../light/themes' as *;
+
+$theme: $material;
```

Comment:
Same as my previous comment. Remove all $theme variables where they're not being used.